zeebe/bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeTaskListenersValidationTest.java

```diff
@@ -59,4 +59,21 @@ void testEventTypeNotDefined() {
         process,
         expect(ZeebeTaskListener.class, "Attribute 'eventType' must be present and not empty"));
   }
+
+  @Test
+  @DisplayName("task listener with unsupported `eventType` property")
+  void testEventTypeNotSupported() {
+    // given
+    final BpmnModelInstance process =
+        Bpmn.readModelFromStream(
+            ReflectUtil.getResourceAsStream(
+                "io/camunda/zeebe/model/bpmn/validation/ZeebeTaskListenersValidationTest.testEventTypeNotSupported.bpmn"));
```

Comment:
🔧 Since the `bpmn-model` Java API supports all planned listener types, you could simplify this test by creating the model programmatically. This would make the test easier to maintain and eliminate the need for a static BPMN file.

Additionally, you can make this test parametrized to dynamically verify all unsupported `eventType` values. Here's an example:

```java
@DisplayName("task listener with unsupported `eventType` property")
@ParameterizedTest(name = "unsupported event type: \''{0}\''")
@EnumSource(
    value = ZeebeTaskListenerEventType.class,
    mode = EnumSource.Mode.EXCLUDE,
    // supported event types
    names = {"assignment", "complete"})
void testEventTypeNotSupported(final ZeebeTaskListenerEventType unsupportedEventType) {
  // given
  final BpmnModelInstance process =
      Bpmn.createExecutableProcess("process")
          .startEvent()
          .userTask(
              "my_user_task",
              ut ->
                  ut.zeebeUserTask()
                      .zeebeTaskListener(
                          l -> l.eventType(unsupportedEventType).type("my_listener")))
          .endEvent()
          .done();

  // when/then
  ProcessValidationUtil.assertThatProcessHasViolations(
      process,
      expect(
          ZeebeTaskListener.class,
          String.format(
              "Task listener event type '%s' is not supported. Currently, only assignment, complete event types are supported.",
              unsupportedEventType)));
}
```

### Benefits:
- 💭 Removes the need for managing a static BPMN file.
- 💭 Covers all unsupported `eventType` values in one test.
- 💭 Adding new supported event types would only require updating the `name` parameter in the `@EnumSource` annotation.

Let me know what you think! 😊